### ISSUE TITLE
Error CS0006 Metadata file could not be found

### ISSUE THREAD

**Issue by owidat** [Microsoft/PartsUnlimited#141]
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0006	Metadata file 'C:\Users\user1\Source\Repos\PartsUnlimited\src\PartsUnlimitedWebsite\bin\Debug\netcoreapp2.0\PartsUnlimitedWebsite.dll' could not be found	PartsUnlimited.UnitTests	C:\Users\user1\Source\Repos\PartsUnlimited\test\PartsUnlimited.UnitTests\CSC	1	Active

Severity	Code	Description	Project	File	Line	Suppression State
Error		Missing binding C:\Users\user1\Source\Repos\PartsUnlimited\src\PartsUnlimitedWebsite\node_modules\grunt-sass\node_modules\node-sass\vendor\win32-x64-48\binding.node	PartsUnlimitedWebsite	C:\Users\user1\Source\Repos\PartsUnlimited\src\PartsUnlimitedWebsite\EXEC	1	

Severity	Code	Description	Project	File	Line	Suppression State
Warning		Task "sass" not found. Use --force to continue.	PartsUnlimitedWebsite	C:\Users\user1\Source\Repos\PartsUnlimited\src\PartsUnlimitedWebsite\EXEC	1	

Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB3073	The command "grunt" exited with code 3.	PartsUnlimitedWebsite	C:\Users\user1\Source\Repos\PartsUnlimited\src\PartsUnlimitedWebsite\PartsUnlimitedWebsite.csproj	41

**Comment by eamonnk** [comment]
Hi, 

From the errors it sounds like you are trying to build the PartsUnlimited app in VS. Those errors may be related to installed packages that need to be updated. 

Can you look at the task steps here [PartsUnlimited Setup with Visual Studio](http://microsoft.github.io/PartsUnlimited/pandp/200.1x-PandP-PUsetupwithVS2017.html). There is a **Build Errors and Solutions** section in there which will help with this error and other similar potential issues.  

Sure let us know if the steps in the link above provide a solution for you or if not let us know what lab task you are stepping through and where you are encountering the error.

Thanks,
Eamonn

**Comment by eamonnk** [comment]
Hi Robert, 

Have you tried stepping through the Build Errors and Solutions section in Task 3, in the lab steps in the link above?

Thanks,
Eamonn

**Comment by RobertCGouge** [comment]
Yes, I verified I am running Node 6.12.3, I've uninstalled and reinstalled Node, Visual studio, node-sass, and I ran git cleanup.  I also removed everything from my local file system and re-cloned into a new directory.

**Comment by eamonnk** [comment]
Grand, I'll run through it and have a look, will let you know later today.

Thanks,
Eamonn

**Comment by eamonnk** [comment]
Hi Again, 

Sorry was delayed getting to this, but just ran through it and was able to build successfully. So functionally it should be possible to get it to build successfully and I don't see anything thats fundamentally changed that would cause a failure. Can you have another look at the output tab in the build output and see exactly what message it is giving, I added an additional screenshot to the lab steps for the output which gives the details you get when you get a missing website.dll  missing binding errors, and the steps then to rebuild the node-sass to address them, 

- Open up a cmd or Powershell window as administrator, go to the repo where the PartsUnlimitedWebsite is located i.e. C:\<repository path>\PartsUnlimited\src\PartsUnlimitedWebsite
- then run the below command and try again to build the solution.
- npm rebuild node-sass 

Can you have a look at that output message and try those workaround steps ?

Thanks,
Eamonn

**Comment by RobertCGouge** [comment]
I am not sure what the issue was.  I tried on a machine that had never had Visual Studio or Node installed and it worked fine and I was able to go through the lab.  I'm going to blame my machine.

**Comment by cb1947** [comment]
Had the same error..finally got it resolved due to grunt issue
used the following command to install grunt

npm install -g grunt-cli

from the following article
https://github.com/gruntjs/grunt/issues/774

no more build errors
